### PR TITLE
navbar design updated and About page UI improved

### PR DESCRIPTION
**Pull request by enessozdemir**

### CODE REVIEW

**Comment by AryanNanda17** [comment]
@enessozdemir, please share ss and it would be awesome if you would star this repo.

**Comment by enessozdemir** [comment]
Ekran Resmi 2024-10-20 15 15 36

**Comment by ParamThakkar123** [comment]
This looks way better than what we have currently

**Comment by ParamThakkar123** [comment]
@enessozdemir There are some merge conflicts here. Resolve them and make a push. We will merge them

**Comment by AryanNanda17** [comment]
@ParamThakkar123, current navbar design is good I feel. WDYT?

**Comment by ParamThakkar123** [comment]
> @ParamThakkar123, current navbar design is good I feel. WDYT?

Yes. But it doesn't look that stylish though 😅 . But the glassmorphism we added is great indeed.

**Comment by ParamThakkar123** [comment]
@enessozdemir ensure that the glassmorphism is preserved in the Navbar

**Comment by ParamThakkar123** [comment]
> @ParamThakkar123, current navbar design is good I feel. WDYT?

Actually I just went through the Navbar right now and I think our current design is better. When I compared both. Sorry to change my decision. But I would just like to revert what I said 😅.

**Comment by ParamThakkar123** [comment]
@enessozdemir keep the navbar design as it is. Please don't change it. I think the current design is good. Just push the about page. No need to change the navbar

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
+/* eslint-disable no-unused-vars */
```
Comment (ParamThakkar123) [inline src/components/Navbar.jsx:1]: We don't need any changes to the navbar file. Keep it the same please

**Pull request merged**